src/cljs/athens/electron.cljs

```diff
@@ -323,6 +324,55 @@
   ;; open or create a new starter db
 
   ;; Watch filesystem, e.g. in case db is updated via Dropbox sync
+  
+  (def schemaTest
+  {:schema/version {}
+   :block/uid      {:db/unique :db.unique/identity}
+   :node/title     {:db/unique :db.unique/identity}
+   :attrs/lookup   {:db/cardinality :db.cardinality/many}
+   :block/children {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}
+   :block/refs     {:db/cardinality :db.cardinality/many
+                    :db/valueType :db.type/ref}}
+  )
+  
+  ;; Create an empty Db for test 
+  (def test-db
+    (-> (d/empty-db schemaTest))
+  )
+  
+  ;; Update index.transit
+  (defn writeDbIndex [file filepath]
+    (let [w  (.. fs (writeFile filepath file (fn [e](dispatch [:boot/desktop]))))])
```

Comment:
does this pass cljstyle? I'd expect cljstyle to fix the missing space after `(fn [e]`

Reply:
done